api/tests/unit/organisations/test_unit_organisation_views.py

```diff
@@ -1340,3 +1340,37 @@ def test_list_my_groups(organisation, api_client):
         "id": user_permission_group_1.id,
         "name": user_permission_group_1.name,
     }
+
+
+def test_when_subscription_is_cancelled_then_remove_all_but_the_first_user(
+    staff_client, admin_client, subscription, organisation
```

Comment:
typing please :)

Reply:
Nice catch! Let's get typing into the checklist when opening PRs. It's hard to remember to do it sometimes.

Reply:
Ok this is handled.